tests/test_core.py

```diff
@@ -986,6 +987,32 @@ def test_partial_method_lots_of_args(self):
         )
         self.assertEqual(buf.value.decode("utf-8"), pystring)
 
+    def test_objcmethod_objcpartialmethod_objcboundmethod_repr(self):
```

Comment:
No need to be *quite* so explicit in the test name:
```suggestion
    def test_objcmethod_repr(self):
```